icms_version.php

```diff
@@ -136,33 +136,23 @@
 
 $modversion['config'][1] = array(
   'name' => 'tager_groups',
-  'title' => '_MI_IMTAGGING_TAGERGR',
-  'description' => '_MI_IMTAGGING_TAGERGRDSC',
+  'title' => _MI_IMTAGGING_TAGERGR,
+  'description' => _MI_IMTAGGING_TAGERGRDSC,
   'formtype' => 'select_multi',
   'valuetype' => 'array',
   'options' => $select_groups_options,
   'default' =>  '1');
 
 $modversion['config'][] = array(
   'name' => 'tags_limit',
-  'title' => '_MI_IMTAGGING_LIMIT',
-  'description' => '_MI_IMTAGGING_LIMITDSC',
+  'title' => _MI_IMTAGGING_LIMIT,
+  'description' => _MI_IMTAGGING_LIMITDSC,
   'formtype' => 'textbox',
   'valuetype' => 'text',
   'default' => 5);
 
```

Comment:
When I added config items to imBlogging, the language constants did need to be enclosed in quotes to be treated as a constant.

Reply:
that is correct, error on my part.